ext/cairo/rb_cairo_font_face.c

```diff
@@ -39,6 +39,11 @@ static ID cr_id_at_need_clusters;
 static ID cr_id_at_need_cluster_flags;
 #endif
 
+#if CAIRO_HAS_FT_FONT
+static FT_Library library;
```

Comment:
If we use static variable, we can't use two or more FreeType based fonts at once. Is it intended?